<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LifeStuffLogin</class>
 <widget class="QWidget" name="LifeStuffLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>LifeStuff</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/32/ms_icon_blue.png</normaloff>:/icons/32/ms_icon_blue.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#TopFrame {
  border-image :url(:/background/login_background.png) 0 0 0 0 stretch stretch;
  font: bold;	
}

QMainWindow {
background-color: white;
color: black;
}

QLineEdit { 
border-width: 1px; 
border-style: inset; 
border-color: #5593d7; 
border-radius: 5px; 
background:lightgrey;
color: grey; 
selection-background-color: ltblue; 
}

QLineEdit:focus{
	background: white;
	color: black;
}





</string>
  </property>
  <widget class="QFrame" name="TopFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>171</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>285</width>
     <height>140</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="progress_label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>351</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progress_bar">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>376</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>360</y>
     <width>105</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../resources/resources.qrc">:/background/maidsafe_logo.png</pixmap>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignHCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>183</y>
     <width>281</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="info_label">
      <property name="text">
       <string>Please enter your credentials</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="UserEdit">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
      <property name="maxLength">
       <number>255</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PinEdit">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>PIN</string>
      </property>
      <property name="maxLength">
       <number>4</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PassEdit">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
      <property name="maxLength">
       <number>32</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="createBtn">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Create</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearBtn">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loginBtn">
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>&amp;Login</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
